common/api/ui-framework.api.md

```diff
@@ -7627,7 +7627,7 @@ export const withSafeArea: <P extends InjectedWithSafeAreaProps, C>(Component: R
 };
 
 // @public
-export const withStatusFieldProps: <P extends StatusFieldProps, C>(Component: React.JSXElementConstructor<P> & C) => (props: JSX.LibraryManagedAttributes<C, Omit<P, "isInFooterMode" | "openWidget" | "onOpenWidget">>) => JSX.Element;

```

Comment:
Looks like you need to `rush clean && rush rebuild && rush extract-api` because your changes should not have touched many of these api.md files.

Reply:
Thanks, that seemed to clean those up

Reply:
@bsteinbk resolving this as it looks like all the extract-api issues are ironed out.